6.2_Bibliotecas_JavaScript_e_Frameworks_CSS/exercises/6_2/js/script.js

```diff
@@ -0,0 +1,59 @@
+
+// Função de criar e popular a div
+function check() {
+  document.getElementById('print').appendChild(document.createElement('p')).innerHTML = document.getElementById('name').value;
+  document.getElementById('print').appendChild(document.createElement('p')).innerHTML = document.getElementById('email').value;
+  document.getElementById('print').appendChild(document.createElement('p')).innerHTML = document.getElementById('cpf').value;
+  document.getElementById('print').appendChild(document.createElement('p')).innerHTML = document.getElementById('address').value;
+  document.getElementById('print').appendChild(document.createElement('p')).innerHTML = document.getElementById('city').value;
+  document.getElementById('print').appendChild(document.createElement('p')).innerHTML = document.getElementById('state').value;
+  document.getElementById('print').appendChild(document.createElement('p')).innerHTML = document.getElementById('textArea').value;
+  document.getElementById('print').appendChild(document.createElement('p')).innerHTML = document.getElementById('role').value;
+  document.getElementById('print').appendChild(document.createElement('p')).innerHTML = document.getElementById('roleDesc').value;
+  document.getElementById('print').appendChild(document.createElement('p')).innerHTML = document.getElementById('startDate').value;
+}
+
+// Função de remover a div
+clearDiv() {
+  document.getElementById('print').remove(appendChild(document.querySelectorAll('p')))
+}
+
+
+//Bibliotecas externas
+//DatePickerX
+document.getElementById('startDate').DatePickerX.init();
+
+//JustValidate
+new window.JustValidate('.js-form');
+
+new JustValidate('.js-form', {
+  rules: {
+    email: {
+      required: true,
+      email: true
+    },
+  messages: {
+    email: {
+      required: 'O campo e-mail é obrigatório'
+      email: 'O email digitado não é válido'
+    }
+  }
+  } 
+})
+
+new window.JustValidate('.js-form', {
```

Comment:
aqui está certinho. Só explicando melhor o que é cada coisa:

Em `rules` você vai colocar quais campos você quer validar, baseado na propriedade `name` do input.

Em `messages` você vai colocar a mensagem de erro pra cada um dos campos.

A função `submitHandler` vai ser chamada quando acontecer o `submit` do form, se não houver nenhum erro de validação. Nela você vai pegar as informações que a pessoa usuária inseriu e tratar como for necessário no seu código.